core/common/src/thrift/file_system_master.thrift

```diff
@@ -25,20 +27,16 @@ struct CreateDirectoryTOptions {
   2: optional bool recursive
   3: optional bool allowExists
   4: optional i16 mode
-  5: optional i64 ttl
-  6: optional common.TTtlAction ttlAction
-  7: optional FileSystemMasterCommonTOptions commonOptions
+  5: optional FileSystemMasterCommonTOptions commonOptions
```

Comment:
to keep backward compatibility, still have `commonOptions` with index 7 and let's not remove index 5 and 6 but simply rename them and comment the deprecation.

```
5: optional i64 ttlNotUsed // deprecated from 1.8
6: optional common.TTtlAction ttlActionNotUsed // deprecated from 1.8
7: optional FileSystemMasterCommonTOptions commonOptions
```